### ISSUE TITLE
How do I properly continue finetuning a checkpoint?

### ISSUE THREAD

**Issue by wickedsprout** [PaddlePaddle/ERNIE#417]
When I attempt to continue finetuning using checkpoint (that was the result of another finetuning procedure), the model stagnates after a while. Basically, I get identical predictions after every epoch. Is this a known problem, maybe I'm doing something incorrectly?
To initialize finetuning with a checkpoint, I use the "--init_checkpoint" argument.

**Comment by oyxuan-11** [comment]
Hi, you can try "--init_pretraining_params" to continue fine-tuning  a checkpoint.
when you use the '--init_checkpoint' argument, you will load the learning rate parameters in your checkpoint model, and the learning rate might be  a very small value(maybe 0) because of the learning rate algorithm we used. That's the reason why you get same results in every epoch.

**Comment by wickedsprout** [comment]
I see. Thank you!
